<?xml version="1.0"?>
<launch>

	<!-- Start the youbot -->
	<include file="$(find cleaner_alpha)/launch/youbot_driver.launch" />

	<!-- Upload youbot model to parameter server --> 
	<!-- TODO: copy youbot.urdf.xacro to this package -->
	<param name="robot_description" command="$(find xacro)/xacro.py $(find grab_demo)/robots/youbot.urdf.xacro" />

	<!-- start laser_assembler -->
	<node type="laser_scan_assembler" pkg="laser_assembler" name="laser_assembler">
		<remap from ="cloud" to="laser" />
		<param name="max_clouds" type="int" value="200" />
		<param name="fixed_frame" type="string" value="base_link" />
	</node>

	<!-- start object finder -->
	<param name="debugging" type="boolean" value="false" />
	<node type="object_finder_2d" pkg="object_finder_2d" name="object_finder_2d" />

</launch>
